extensions/data-transfer/portability-data-transfer-rememberthemilk/src/main/java/org/dataportabilityproject/transfer/rememberthemilk/model/tasks/TaskAddResponse.java

```diff
@@ -15,10 +15,13 @@
  */
 package org.dataportabilityproject.transfer.rememberthemilk.model.tasks;
 
-import com.google.api.client.util.Key;
+import com.fasterxml.jackson.xml.annotate.JacksonXmlProperty;
 
 public class TaskAddResponse extends RememberTheMilkResponse {
 
-  @Key("list")
-  public TaskList taskList;
+  @JacksonXmlProperty(localName = "transaction")
+  public Transaction transaction;
```

Comment:
Should we add a comment here. I'm not sure what transaction means in this context.

Reply:
Done.